Where is `dist` folder and  `web3.min.js`?

After `npm install --save web3`, I only see `packages` and `src` foldesr inside `node_modules/web3`. Where is the `dist` folder containing `web3.min.js`?

after running `npm install web3@1.0.0-beta2 --save` i'm missing utils functions.

i went into `/node_modules/web3/` and ran `npm install` and `npm run-script build` but i get the install error:
`Error: Cannot find module 'web3-core' from '[my_project_path]/node_modules/web3/packages/web3-net/src'`

--- a/.gitignore
+++ b/.gitignore
@@ -14,4 +14,5 @@ bower_components
 .npm/
 .vscode/
 dist/
+packages/web3/dist/
 !./dist/web3.min.js

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -56,6 +56,7 @@ Released with 1.0.0-beta.37 code base.
 - getNetworkType method extended with Görli testnet (#3095)
 - supportsSubscriptions method added to providers (#3116)
 - Add `eth.getChainId` method (#3113)
+- Minified file added to web3 package (#3131)
 
 ### Fixed
 

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -20,6 +20,7 @@ var replace = require('gulp-replace');
 var exec = require('child_process').exec;
 
 var DEST = path.join(__dirname, 'dist/');
+var WEB3_PACKAGE_DEST = path.join(__dirname, 'packages/web3/dist');
 
 var packages = [{
     fileName: 'web3',
@@ -189,6 +190,7 @@ packages.forEach(function (pckg, i) {
             .pipe(streamify(uglify(ugliyOptions)))
             .on('error', function (err) { console.error(err); })
             .pipe(rename(pckg.fileName + '.min.js'))
+            .pipe(gulp.dest(WEB3_PACKAGE_DEST))
             .pipe(gulp.dest(DEST));
     }));
 });

--- a/packages/web3/package.json
+++ b/packages/web3/package.json
@@ -8,6 +8,7 @@
         "node": ">=8.0.0"
     },
     "main": "src/index.js",
+    "browser": "dist/web3.min.js",
     "bugs": {
         "url": "https://github.com/ethereum/web3.js/issues"
     },

